server/src/main/java/org/elasticsearch/inference/InferenceService.java

```diff
@@ -97,6 +97,17 @@ default void stop(String modelId, ActionListener<Boolean> listener) {
         listener.onResponse(true);
     }
 
+    /**
+     * Put the model definition (if applicable)
+     * The main purpose of this function is to download ELSER
+     * The default action does nothing except acknowledge the request (true).
+     * @param modelVariant The configuration of the model variant to be downloaded
+     * @param listener The listener
+     */
+    default void putModel(Model modelVariant, ActionListener<Boolean> listener) {
```

Comment:
Should we leverage the `start` method that's in this interface for downloading the model instead of adding a new one?

Reply:
I think we could use `start`, although I am unsure about the benefit. I suppose it would decrease the complexity of the interface, and it is a bit weird that `startModel` in `TransportPutInferenceModelAction` calls two methods in the InferenceService. However, I think that this also represents the underlying business logic (that start is both starting an inference model and putting a trained model definition). I think that creating a second method in the interface has the benefit of splitting up these two objectives into separate methods in the `ElserMlNodeService` (and other `InferenceService`s), which represent the two API calls we are making to support the business logic. 

So, I can't really imagine why it would be better to avoid creating another method in this interface, but please let me know if I missed it!